Scripts/Terminology-Install-Utilities.psm1

```diff
@@ -520,29 +576,65 @@ function Invoke-PostToMds {
         [ValidateNotNullOrEmpty()]
         [PSCustomObject] $Config
     )
-
-    # TODO THIS NEEDS TO CHANGE AFTER THE FABRIC UPDATES!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-    $authToken = Read-Host "Log into Atlas was a DosAdmin, and copy the access token from one of the auth headers to MDS and copy it here. This is a temporary workaround"
+    Write-DosMessage -Level "Information" "Reading file $path"
     $dataMartJson = Get-Content -Raw -Path $path
     $headers = @{"Content-Type" = "application/json"}
-    $headers.Add("Authorization", "Bearer $authToken")
+    $headers.Add("Authorization", "Bearer $($Config.accessToken)")
 
     try {
-        Write-DosMessage -Level "Information" -Message "Starting to create metadata for $name"
-        $response = Invoke-RestMethod -Uri "$($Config.mdsServiceUrl)/DataMarts" -Method POST -Body $dataMartJson -Headers $headers -UseBasicParsing
-        Write-DosMessage -Level "Information" -Message "Completed creating metadata for $name"
-        return $response.Id
+        Write-DosMessage -Level "Information" -Message "Posting json metadata in $path for $name in mds $($Config.mdsServiceUrl)/DataMarts.`nThis may take several minutes"
```

Comment:
We should capitalize MDS (and possibly JSON)

Reply:
Or maybe expand it to "metadata service"

Reply:
👍